### PR TITLE
refactor: rename platformOptions -> platformPrOptions

### PR DESCRIPTION
**Pull request by RahulGautamSingh**
## Changes
Change the existing internal variable  `platformOptions` to `platformPrOptions` so that we do not confused it with the to-be-added new option `platformOptons`

## Context
Ref: https://github.com/renovatebot/renovate/issues/29716#issuecomment-2227302251

## Documentation (please check one with an [x])

- [ ] I have updated the documentation, or
- [x] No documentation update is required

## How I've tested my work (please select one)

I have verified these changes via:

- [x] Code inspection only, or
- [ ] Newly added/modified unit tests, or
- [ ] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

**Comment by rarkins** [comment]
The new option will need a new name

**Comment by RahulGautamSingh** [comment]
> The new option will need a new name

Should I close this or is this still okay?

**Comment by rarkins** [comment]
Actually it's ok if the new one isn't publicly exposed for config yet

**Review by viceice — DISMISSED** [review]

File: lib/modules/platform/types.ts [hunk 1]
```diff
@@ -112,13 +112,13 @@ export interface CreatePRConfig {
   prTitle: string;
   prBody: string;
   labels?: string[] | null;
-  platformOptions?: PlatformPrOptions;
+  platformPrOptions?: PlatformPrOptions;
```
Comment (viceice) [inline lib/modules/platform/types.ts:115]: `prPlatformOptions` sounds nicer 🤷‍♂️

**Review by viceice — APPROVED** [review]

File: lib/modules/platform/types.ts (hunk 1 above)
Comment (RahulGautamSingh) [reply lib/modules/platform/types.ts:115]: I don't hear it 🙂

**Comment by RahulGautamSingh** [comment]
Could we merge this, need this refactor in an upcoming PR?

**Pull request merged**